### DISCUSSION TITLE
useExtendedRemote support on 1.10.7?

### DISCUSSION THREAD

**Discussion by fbs2016** [microsoft/vscode-cpptools#9491] [General]
I see that the option "useExtendedRemote" has been commit in https://github.com/microsoft/MIEngine/pull/1260, but can't set it in cpptools latest release version 1.10.7 or pre-release 1.11.0. Do we support it now?

Thanks

**Comment by sean-mcmanus** [comment]
@fbs2016 This is available in 1.11.3 (pre-release): https://github.com/microsoft/vscode-cpptools/releases/tag/v1.11.3
